docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,299 @@
+the regularly published Helm chart. The Helm chart contains all the custom
+resources and controllers needed to deploy and configure Crossplane.
+
+See [Install] and [Configure] docs for installing alternate versions and more
+detailed instructions.
+
+## Get a Kubernetes Cluster
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#setup-mac-brew" data-toggle="tab">macOS via Homebrew</a></li>
+<li><a href="#setup-mac-linux" data-toggle="tab">macOS / Linux</a></li>
+<li><a href="#setup-windows" data-toggle="tab">Windows</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="setup-mac-brew" markdown="1">
+For macOS via Homebrew use the following:
+
+```
+brew upgrade
+brew install kind
+brew install kubectl
+brew install helm
+
+kind create cluster --image kindest/node:v1.17.5 --wait 5m
+```
+
+</div>
+<div class="tab-pane fade" id="setup-mac-linux" markdown="1">
+For macOS / Linux use the following:
+
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+
+</div>
+<div class="tab-pane fade" id="setup-windows" markdown="1">
+For Windows use the following:
+
+* [Kubernetes cluster]
+  * [Kind]
+  * [Minikube], minimum version `v0.28+`
+  * etc.
+
+* [Helm], minimum version `v2.12.0+`.
+  * For Helm 2, make sure Tiller is initialized with sufficient permissions to
+    work on `crossplane-system` namespace.
+
+</div>
+</div>
+
+## Install Crossplane
+<ul class="nav nav-tabs">
+<li class="active"><a href="#install-tab-helm3" data-toggle="tab">Helm 3 (alpha)</a></li>
+<li><a href="#install-tab-helm2" data-toggle="tab">Helm 2 (alpha)</a></li>
+<li><a href="#install-tab-helm3-master" data-toggle="tab">Helm 3 (master)</a></li>
+<li><a href="#install-tab-helm2-master" data-toggle="tab">Helm 2 (master)</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="install-tab-helm3" markdown="1">
+Use Helm 3 to install the latest official `alpha` release of Crossplane, suitable for community use and testing:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm2" markdown="1">
+Use Helm 2 to install the latest official `alpha` release of Crossplane, suitable for community use and testing:
+
+```
+kubectl create namespace crossplane-system
+
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+</div>
+<div class="tab-pane fade" id="install-tab-helm3-master" markdown="1">
+Use Helm 3 to install the latest `master` pre-release version of Crossplane:
```

Comment:
Are we planning on keeping the existing, more detailed install and configure docs? If so can we assume folks will be using alpha for the getting started guide and leave the detail on how to install pre-release master versions (which are very likely to break this guide) to the more detailed variants?

Reply:
I'm hoping that detailed install and config docs can go away with this.